### PR TITLE
[fix #103] Change 'explore graph paths' option description

### LINKED ISSUE #103: Fst2List option is unclear
Fst2List recursive exploration option description makes the behaviour of each choice unclear.
* "_Only paths_" doesn't imply clearly the recursive exploration.
* "_Do not explore subgraphs recursively_" doesn't explain what is the expected output.

issue-100

### PR DESCRIPTION
**Pull request by Kalkhas**
Change the options' descriptions to explicit what they actually do.

### CODE REVIEW

File: unitex/src/fr/umlv/unitex/frames/GraphPathFrame.java [hunk 1]
```diff
@@ -133,9 +133,9 @@ private JPanel constructTopPanel() {
 		final JPanel top1 = new JPanel(new FlowLayout(FlowLayout.LEFT));
 		final ButtonGroup pathWithSubGraph = new ButtonGroup();
 		
-		onlyPaths = new JRadioButton("Only paths", true);
+		onlyPaths = new JRadioButton("Explore recursively", true);
```
Comment (martinec) [inline unitex/src/fr/umlv/unitex/frames/GraphPathFrame.java]: The better would be to prepend a label "Calls to subgraphs:" before the the two option buttons, something like this:

`Calls to subgraphs: (o) Explore calls recursively ()  Print calls without exploring it`

This is consistent with the stated in the User Manual, section 6.5, Exploring grammar paths.

Comment (Kalkhas) [reply unitex/src/fr/umlv/unitex/frames/GraphPathFrame.java]: Do you also want me to rename the choices ?

Comment (martinec) [reply unitex/src/fr/umlv/unitex/frames/GraphPathFrame.java]: Yea is that, `Explore calls recursively` for `Only paths` and `Print calls without exploring it` for  `Do not explore subgraphs recursively`. The visual result would be something like this:

`Calls to subgraphs: (o) Explore calls recursively () Print calls without exploring it`

Thank your for your help !

Comment (Kalkhas) [reply unitex/src/fr/umlv/unitex/frames/GraphPathFrame.java]: We came up with a new solution we thought was more clear with @eric-laporte.
It looks like this: 

explore_graph_ui_01

**Comment by eric-laporte** [comment]
OK

**Pull request merged**